Runtime/Game/Requests/CatalogRequests.cs

```diff
@@ -511,7 +582,8 @@ public LootLockerInlinedCatalogEntry[] GetLootLockerInlinedCatalogEntries()
                     LootLockerCatalogEntryEntityKind.asset == entityKind && asset_details.ContainsKey(catalogListingID) ? asset_details[catalogListingID] : null, 
                     LootLockerCatalogEntryEntityKind.progression_points == entityKind && progression_points_details.ContainsKey(catalogListingID) ? progression_points_details[catalogListingID] : null, 
                     LootLockerCatalogEntryEntityKind.progression_reset == entityKind && progression_resets_details.ContainsKey(catalogListingID) ? progression_resets_details[catalogListingID] : null, 
-                    LootLockerCatalogEntryEntityKind.currency == entityKind && currency_details.ContainsKey(catalogListingID) ? currency_details[catalogListingID] : null
+                    LootLockerCatalogEntryEntityKind.currency == entityKind && currency_details.ContainsKey(catalogListingID) ? currency_details[catalogListingID] : null,
+                    LootLockerCatalogEntryEntityKind.group == entityKind && group_details.ContainsKey(catalogListingID) ? group_details[catalogListingID] : null
```

Comment:
We need to chat about this. I have a vague recollection about this not going to work. It was something about us now needing a combined key of catalog_listing_id and something else. So let's chat about it.